modules/psap-driver-toolkit.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/psap-driver-toolkit.adoc
+
+[id="about-driver-toolkit"]
+= About the Driver Toolkit
+
+[discrete]
+== Background
+The Driver Toolkit is a container image in the {product-title} payload which is meant to be used as a base image on which to build driver containers. The Driver Toolkit image contains the kernel packages commonly required as dependencies to build or install kernel modules as well as a few tools needed in driver containers. The version of these packages will match the kernel version running on the RHCOS nodes in the corresponding {product-title} release. 
+
+Driver containers are container images used for building and deploying out-of-tree kernel modules and drivers on container OSs like Red Hat Enterprise Linux CoreOS (RHCOS). Kernel modules and drivers are software libraries running with a high level of privilege in the operating system kernel. They extend the kernel functionalities or provide the hardware-specific code required to control new devices.  Examples include hardware devices like FPGAs or GPUs, and software defined storage (SDS) solutions like Lustre parallel filesystem, which all require kernel modules on client machines. Driver containers are the first layer of the software stack used to enable these technologies on Kubernetes.
+
+The list of kernel packages in the Driver Toolkit includes the following and their dependencies:
+
+* kernel-core
+* kernel-devel
+* kernel-headers
+* kernel-modules
+* kernel-modules-extra
+
+In addition, the Driver Toolkit also includes the corresponding real-time kernel packages:
+
+* kernel-rt-core
+* kernel-rt-devel
+* kernel-rt-modules
+* kernel-rt-modules-extra
+
+The driver toolkit also has several tools which are commonly needed to build and install kernel modules, including:
+
+* elfutils-libelf-devel
+* kmod
+* binutilskabi-dw
+* kernel-abi-whitelists
+* dependencies for the above
+
+[discrete]
+== Purpose
+Before the Driver Toolkit, kernel packages could be installed in a Pod or BuildConfig on OpenShift using link:https://www.openshift.com/blog/how-to-use-entitled-image-builds-to-build-drivercontainers-with-ubi-on-openshift[entitled builds] or by installing from the kernel RPMs in the hosts machine-os-content. The Driver Toolkit simplifies the process by removing the entitlement step, and allows us to avoid the privileged operation of accessing the machine-os-content in a Pod. The Driver Toolkit can also be used by partners who have access to pre-released OpenShift versions to prebuild driver-containers for their hardware devices for future OpenShift releases.
```

Comment:
Prior to the Driver Toolkit's existence, you could install kernel packages in a pod or build config on {product-title} using

Reply:
host's `machine-os-content` package  (is it a package?)

s/allows us to avoid/avoids 

s/Pod/pod

Reply:
s/OpenShift/{product-title}